Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -273,6 +274,15 @@ def main():
     pr.create_issue_comment(body)
     print(f'{t.cyan}Created welcome comment{t.normal}')
 
+    print('contributors.md section')
+    print(f'pack path: {pr_files[0]}')
+    ver = get_pack_metadata(pr_files[0]).get('currentVersion')
+    print(f'version is: {ver}')
+    contributors_body = f'Hi @{pr.user.login}, Thanks for contributing to a Cortex XSOAR supported pack. To receive ' \
+                        f'credit for your generous contribution please follow this [link]' \
+                        f'(https://xsoar.pan.dev/docs/packs/packs-format#contributorsjson).'
```

Comment:
See the if in line 272 that checks if the contributor contributed through the UI or not and changes the message accordingly. If it is from the UI add that they can ask the reviewer to add the file for them.